src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php

```diff
@@ -311,9 +311,15 @@ private function getProperties(UploadedFile $uploadedFile)
                 } catch (\RuntimeException $e) {
                     // Exception is thrown if the dimension could not be extracted
                 }
+            } catch (ExecutableNotFoundException $e) {
+                // Exception is thrown if ffmpeg is not installed -> video properties are not set
+            }
+        } elseif (\fnmatch('image/*', $mimeType)) {
+            $dimensions = \getimagesize($uploadedFile->getPathname());
```

Comment:
As written in the comment above `getimagesize` is a gd specific function to support imagick, gmagick or vips we need to replace this with the `imagine` service which should be injected as additonal optional service:

```php
if ($this->imagine) {
    $image = $this->imagine->open(..);
    $properties['width'] = $image->getSize()->getHeight();
    $properties['height'] = $image->getSize()->getWidth();
}
```

Reply:
All modifications done, sorry for the delay.